content/about/_index.md

```diff
@@ -46,9 +67,18 @@ Google to power use cases from microservices to the "last mile" of computing
 For more background on why we created gRPC, see the [gRPC Motivation and Design
 Principles](/blog/principles/) on the [gRPC blog](/blog/).
 
-{{< note >}}
-  <a name="officially-supported-languages-and-platforms"></a>
+{{< alert title="Note" color="info" >}}
```

Comment:
Is `alert` a custom shortcode? If so, can we rename `color` to, say, `kind`, and can the `title` have a default value per `kind`?

Reply:
`alert` is a shortcode coming from Docsy - the doc on it is [here](https://www.docsy.dev/docs/adding-content/shortcodes/#alert) - from looking at the template, it looks like the `title` param is optional, but none will display without it.

Reply:
Ok, then we'll use this as is, thanks.